src/collection-view.js

```diff
@@ -165,27 +165,38 @@ Marionette.CollectionView = Marionette.View.extend({
   reorder: function() {
     var children = this.children;
     var models = this._filteredSortedModels();
-    var modelsChanged = _.find(models, function(model) {
+    var modelsAdded = _.find(models, function(model) {
```

Comment:
:+1: I think the intent was to short circuit when an item was found. `_.some` might be more semantic given that we check it's truthiness below. I can make the change if you guys think it makes it more clear.